src/com/sap/piper/tools/ToolVersion.groovy

```diff
@@ -0,0 +1,27 @@
+package com.sap.piper.tools
+
+
+enum ToolVersion {
+
+    JAVA ('1.8.0', '-version 2>&1'), 
+    MTA ('1.0.6', '-v'),
```

Comment:
I don't think it is possible. There is a clear separation between utils and steps.

Reply:
The point is: we have - also today - several tools depending on java, currently neo and mta, but also mvn, where we do not check the java version at all. Different tools depends on different java versions, in most cases the min version is important. Now we have here - for all the tools dending on java the same version. Now we can say, OK this is a refactoring PR and it was like this before. But since we - with the refactoring move from a more lightweight environment check to some kind of environment check framework I would like to outline that having a single version hard coded in an enum will not fit for future needs ...

Reply:
Note that it checks for a compatible version, that's it the same as a min version.